templates/etc_init_archivematica-storage-service-celery-worker.conf.j2

```diff
@@ -0,0 +1,36 @@
+# This file is managed with ansible
+description "Celery worker for Archivematica Storage Service"
+author "Artefactual Systems"
+
+start on (runlevel [2345] and filesystem)
+stop on runlevel [016]
+
+setuid archivematica
+setgid archivematica
+
+# Try respawn a few times in case of errors, then bug out.
+respawn
+respawn limit 20 5
+
+# Time to wait between TERM and KILL signals (relate this to your task execution time)
+kill timeout 30
+
+# django ENV settings
+{% for key, value in archivematica_src_ss_environment.iteritems() %}
+env {{ key }}={{ value }}
+{% endfor %}
+
+# Celery debug level. Can be: DEBUG, INFO, WARNING, ERROR, CRITICAL, or FATAL.
+env LOGLEVEL=INFO
+
+# Celery's specific app definition (django app folder is in a project/apps/djangoapp dir)
+env APP=storage_service
+
+post-start script
+   echo "archivematica storage service - Celery workers started"
```

Comment:
Interesting! Where does this go? To `/var/log/upstart/archivematica-storage-service-celery-worker.log`? Any specific reason?